js/test/test.js

```diff
@@ -141,25 +176,50 @@ async function test (methodName, exchange, ... args) {
     }
 }
 
-async function testSymbol (exchange, symbol) {
-
-    await test ('loadMarkets', exchange);
-    await test ('fetchCurrencies', exchange);
-    await test ('fetchTicker', exchange, symbol);
-    await test ('fetchTickers', exchange, symbol);
-    await test ('fetchOHLCV', exchange, symbol);
-    await test ('fetchTrades', exchange, symbol);
-
-    if (exchange.id === 'coinbase') {
+async function testSafePublic(methodName, exchange, ...args) {
+    return await testMethod(methodName, exchange, ...args);
+}
 
-        // nothing for now
+async function testSafePrivate(methodName, exchange, ...args) {
```

Comment:
@ttodua I don't think we need  `testSafePrivate` or `testSafePublic` because this method does not care about that is just a safe wrapper, I would leave it as `testSafe` for now